joomla/joomla-cms
# [5.x] Localize own package

Pull Request for Issue # .

### Summary of Changes

- Localise my own gzip package `@dgrammatiko/compress`

### Testing Instructions

Check out this PR
- run `npm i`
- run `npm gzip`

Check that the gzipped files inside the media folder are created
Rename the htaccess.txt and check if the gzipped files are loading correctly

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:
